tdtax/phenomenological.yaml

```diff
@@ -0,0 +1,34 @@
+class: Phenomenological
+tags: [Time series characteristics]
+subclasses:
+  - class: non-variable
+  - class: variable
+  - class: periodic
```

Comment:
periodic is a type of variable -- it should be a subclass. Same with most of the other classes, like dipping and flaring.